cmd/helmExecute_test.go

```diff
@@ -322,27 +325,56 @@ func TestRunHelmDefaultCommand(t *testing.T) {
 			methodLintError:    nil,
 			methodPackageError: nil,
 			methodPublishError: nil,
+			fileUtils:          fileHandlerMock{},
+			// we expect the values file is traversed since parsing and rendering according to cpe template is active
+			assertFunc: func(f fileHandlerMock) error {
+				if len(f.fileExistsCalled) == 1 && f.fileExistsCalled[0] == "/values.yaml" {
```

Comment:
This test fails because FileExists is not called. It seems `RenderValuesTemplate` is false, although the default should be true.
(Btw. e.g. `TemplateEndDelimiter` is also empty in the test, although the default is different)

Reply:
I think in production the [step config is prepared](https://github.com/SAP/jenkins-library/blob/master/cmd/helmExecute_generated.go#L129) (probably also the defaults) and [passed to `helmExecute`](https://github.com/SAP/jenkins-library/blob/master/cmd/helmExecute_generated.go#L198).

In the tests we directly call `runHelmExecute` with the plain (unprepared) `helmExecuteOptions`.

The simplest way to fix the test would be to have a test with `RenderValuesTemplate: true` set.

Reply:
https://github.com/SAP/jenkins-library/pull/4511/commits/18c5723b71c09f9a90dd7411b3ecf85657cf52b8

Reply:
Great, thanks! I wonder if that's how it's done in other unit tests as well...

Reply:
At least some parameters I checked are set in tests explicitly to their default value.